.github/actions/check_artifact_exists/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "check_artifact_exists",
+  "main": "main.js",
+  "dependencies": {
+    "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
+    "adm-zip": "^0.5.2",
+    "filesize": "^6.1.0"
+  }
+}
```

Comment:
When you move the action documentation into this folder, can you reference the official docs or write some words yourself on how to handle changes to dependencies? I assume it'll be something like "change package.json, `npm install`, test, commit `node_modules` folder and `package-lock.json` file" but it's good to have it documented. If this is covered in official GitHub docs then just a link would be enough.

Reply:
Sure. One thing I want to test is if it's better to NOT commit `node_modules` and rely on an extra `npm install` step, but I fear it might make the action more fragile (remember all the infras issues because of npm).

Reply:
Yeah seems like fewer live `npm` calls are best from our experience. Maybe we can move to the GitHub recommended step of checking in the compiled `dist/index.js` file instead of node_modules. But I think we don't need to block on that (or even do that work ourselves).

Reply:
I need to re-verify that